collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/FormHierarchyTest.java

```diff
@@ -227,4 +228,40 @@ public void hierachyView_shouldNotChangeAfterScreenRotation() {
                 .rotateToPortrait(new FormHierarchyPage("Repeat Group"))
                 .checkIfElementInHierarchyMatchesToText("Group Name", 0);
     }
+
+    @Test
+    public void groups_shouldBeVisibleInHierarchyView() {
```

Comment:
Could the length of this be reduced do you think?

Reply:
I don't think so. I guess it was important here to use a form with nested repeats and add a few groups that make the test long. @dbemke again if you know anything about this test please let us know, it is TestCase28 NODK-244